lib/IRGen/IRGen.cpp

```diff
@@ -893,7 +897,7 @@ static void performParallelIRGeneration(IRGenOptions &Opts,
     
     // There must be an output filename for each source file.
     // We ignore additional output filenames.
-    if (OutputIter == Opts.OutputFilenames.end()) {
+    if (OutputIter == outputFilenames.end()) {
       // TODO: Check this already at argument parsing.
```

Comment:
Hey, I think you fixed this TODO by now!

Reply:
Cool! Removed.